src/main/java/me/iblitzkriegi/vixio/util/Util.java

```diff
@@ -174,41 +140,43 @@ public static Message messageFrom(Object input) {
         return null;
     }
 
-    public static boolean botIsConnected(Bot bot, JDA jda){
+    public static boolean botIsConnected(Bot bot, JDA jda) {
         return bot.getJDA() == jda;
     }
 
     public static Guild bindGuild(Bot bot, Guild guild) {
         if (!(guild.getJDA() == bot.getJDA())) {
             return bot.getJDA().getGuildById(guild.getId());
         } else {
-            if (guild != null) {
-                return guild;
-            }
-            return null;
+            return guild;
         }
     }
 
     public static TextChannel bindChannel(Bot bot, TextChannel textChannel) {
         if (!(textChannel.getJDA() == bot.getJDA())) {
             return bot.getJDA().getTextChannelById(textChannel.getId());
         } else {
-            if (textChannel != null) {
-                return textChannel;
-            }
-            return null;
+            return textChannel;
         }
     }
 
+    public static MessageChannel bindChannel(Bot bot, MessageChannel messageChannel) {
```

Comment:
This is supposed to be textchannel. This method needs to be reverted to as it was

Reply:
there is no as it was, this is a seperate method